test/unit/collection.test.js

```diff
@@ -1,29 +0,0 @@
-'use strict';
-
-const EventEmitter = require('events');
-const chai = require('chai');
-const expect = chai.expect;
-const { Db } = require('../../src/db');
-
-class MockTopology extends EventEmitter {
-  constructor() {
-    super();
-  }
-
-  capabilities() {
-    return {};
-  }
-}
-
-describe('Collection', function () {
-  it('should not allow atomic operators for findOneAndReplace', {
```

Comment:
is the behavior this test is tracking covered by some other test? Otherwise, it seems like the more prudent approach would be to try to mock a `MongoClient` with a `MockTopology`

Reply:
I moved the test to `test/functional/find_and_modify.test.js` and just used `withClient` instead of mocking. Is that an acceptable alternative?

Reply:
awesome, yes that's great